.circleci/config.yml

```
@@ -1007,7 +1022,9 @@ jobs:
                       docker login --username="${DOCKERHUB_BOT_USER_NAME}" -p="${DOCKERHUB_BOT_USER_TOKEN}"
 
                       # Push all tags if not dry-run mode
-                      if [ "<< parameters.dry_run >>" == "false" ]; then                    
+                      if [ "<< parameters.dry_run >>" == "false" ]; then
+                        export IMAGE_TAG="${GRAVITEEIO_VERSION_MAJOR}.${GRAVITEEIO_VERSION_MINOR}${DOCKER_TAG_SUFFIX}"
```

It should be the full version number I think
```suggestion
                        export IMAGE_TAG="${GRAVITEEIO_VERSION}${DOCKER_TAG_SUFFIX}"
```